kite-morphlines/kite-morphlines-core/src/main/java/org/kitesdk/morphline/stdlib/HashCommandBuilder.java

```diff
@@ -0,0 +1,124 @@
+
+/*
+ * Copyright 2013 Cloudera Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kitesdk.morphline.stdlib;
+
+import java.security.MessageDigest;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.ListIterator;
+
+import org.apache.commons.codec.binary.Hex;
+import org.apache.commons.codec.binary.StringUtils;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.kitesdk.morphline.api.Command;
+import org.kitesdk.morphline.api.CommandBuilder;
+import org.kitesdk.morphline.api.MorphlineContext;
+import org.kitesdk.morphline.api.Record;
+import org.kitesdk.morphline.base.AbstractCommand;
+import com.typesafe.config.Config;
+
+/**
+ * Command that converts the timestamps in a given field from one of a set of
+ * input date formats (in an input timezone) to an output date format (in an
+ * output timezone), while respecting daylight savings time rules. Provides
+ * reasonable defaults for common use cases.
+ */
+
+public class HashCommandBuilder implements CommandBuilder {
```

Comment:
Builder class should be named "HashDigestBuilder" if the command is named "hashDigest" per convention. Or maybe "GenerateHashDigestBuilder" with command named "generateHashDigest" ala "generateUUID" command?